src/octree/mod.rs

```diff
@@ -92,8 +91,8 @@ fn size_in_pixels(bounding_cube: &Cube, matrix: &Matrix4f, width: i32, height: i
 #[derive(Debug)]
 pub struct OnDiskOctree {
     directory: PathBuf,
-    // Maps from node id to number of points.
-    nodes: HashMap<NodeId, u64>,
+    // Maps from node id to NodeMeta: bounding_box, number of points, PositionEncoding.
```

Comment:
this comment will go out of sync as as NodeMeta changes. It is also now tautologically, as the data type contains the same information.